tests/system/test_client.py

```diff
@@ -2682,6 +2682,7 @@ def test_create_routine(self):
             )
         ]
         routine.body = "return maxValue(arr)"
+        routine.determinism_level = "DETERMINISTIC"
```

Comment:
Should determinism level be an enum?

I can see some dedicated string "enum" classes, e.g. `WriteDisposition` , while some other enum values (as defined by API) are set by just using plain strings, e.g. `RoutineArgument.mode`.

Reply:
Probably **should**. We usually get a few pieces of sample feedback in our cloud.google.com docs if we use raw strings anywhere instead of enum-like constants. It is more to maintain, though.

Reply:
Added an enum-like class similar to e.g. `WriteDisposition` to use instead of raw strings. Used such a class instead of a "real" enum both for consistency, and for keeping the logic unchanged.